src/components/pages/redaction.js

```diff
@@ -1,32 +1,38 @@
 import by from 'thenby';
 import PropTypes from 'prop-types';
-import Restricted from '../restricted.js';
 import routes from '../../utils/routes.js';
 import Logger from '../../common/logger.js';
 import FilterInput from '../filter-input.js';
 import { useUser } from '../user-context.js';
-import { Button, Switch, Tooltip } from 'antd';
+import { Switch, Tabs, Tooltip } from 'antd';
 import { useTranslation } from 'react-i18next';
+import { useRequest } from '../request-context.js';
 import DocumentsTable from '../documents-table.js';
 import SortingSelector from '../sorting-selector.js';
 import CloseIcon from '../icons/general/close-icon.js';
 import DocumentInfoCell from '../document-info-cell.js';
 import { handleApiError } from '../../ui/error-helper.js';
 import LanguageIcon from '../localization/language-icon.js';
 import DuplicateIcon from '../icons/general/duplicate-icon.js';
+import { CheckOutlined, LikeOutlined } from '@ant-design/icons';
 import DocumentMetadataModal from '../document-metadata-modal.js';
 import { useSessionAwareApiClient } from '../../ui/api-helper.js';
-import React, { useCallback, useEffect, useMemo, useState } from 'react';
 import DocumentApiClient from '../../api-clients/document-api-client.js';
 import permissions, { hasUserPermission } from '../../domain/permissions.js';
-import { CheckOutlined, LikeOutlined, PlusOutlined } from '@ant-design/icons';
 import { documentExtendedMetadataShape } from '../../ui/default-prop-types.js';
 import { DOCUMENT_METADATA_MODAL_MODE } from '../document-metadata-modal-utils.js';
+import React, { Fragment, useCallback, useEffect, useMemo, useState } from 'react';
 import ActionButton, { ActionButtonGroup, ACTION_BUTTON_INTENT } from '../action-button.js';
 import { DOCUMENT_ALLOWED_OPEN_CONTRIBUTION, DOC_VIEW_QUERY_PARAM } from '../../domain/constants.js';
 
 const logger = new Logger(import.meta.url);
 
+const TABS = {
+  documents: 'documents'
+};
+
+const determineTab = query => Object.values(TABS).find(val => val === query) || TABS.settings;
```

Comment:
`TABS.settings` does not exist :laughing:

Reply:
Corrected (and improved in copy origin too 😋 )

Reply:
:rofl: